### ISSUE TITLE
TLS 1.0 is not supported – site is inaccessible on Windows XP

### ISSUE THREAD

**Issue by ollietreend** [WordPress/browsehappy#41]
browsehappy.com is inaccessible on Windows XP machines. This is because the HTTPS server is configured to only support the TLS 1.2 protocol, which is not compatible with Windows XP. Windows XP [only supports up to TLS 1.0](https://www.ssllabs.com/ssltest/viewClient.html?name=IE&version=8&platform=XP).

Many HTTPS websites still support TLS 1.0 precisely for this reason – for backwards compatibility, to continue serving traffic to Windows XP clients. For example [Google](https://www.ssllabs.com/ssltest/analyze.html?d=google.co.uk&s=216.58.192.3) and [Facebook](https://www.ssllabs.com/ssltest/analyze.html?d=facebook.com&s=157.240.17.35&latest) (see 'Protocols' under the 'Configuration' heading).

TLS 1.2 is the current, modern, recommended protocol to use. However the target audience of browsehappy.com is likely to be running old software (i.e. Windows XP) which doesn't support this modern cipher. I therefore think this should be taken into consideration, and TLS 1.0 support be added so that users of the oldest technology still have the opportunity to improve their browsing experience. (By installing Firefox on Windows XP, users will gain the ability to browse TLS 1.2 websites.)

There seems to be a clear and genuine user need for supporting the old TLS 1.0 protocol in this case. Alternatively, you should allow the website to be served over HTTP rather than forcing HTTPS.

**Comment by jaywilliams** [comment]
I was testing a website today using Windows XP, and this website completely failed to load due to the TLS configuration, which makes the entire point of redirecting users of old IE browsers to this site pointless, as there's nothing for them to see.

I'd advise offering a HTTP and HTTPS version of the website so older browsers can still access it.

**Comment by dominic-p** [comment]
Yeah, the HTTPS redirect to TLS 1.2 effectively defeats the entire purpose of the site as far as I can tell.

**Comment by jaywilliams** [comment]
Since this issue hasn't been resolved, does someone have an alternative to link to provide visitors of old browsers?

**Comment by dominic-p** [comment]
@jaywilliams I've started linking to http://outdatedbrowser.com. That site allows plain, HTTP access.
